plugins/ltac/tauto.ml

```diff
@@ -253,6 +253,20 @@ let with_flags flags _ ist =
   let ist = { ist with lfun = Id.Map.add x.CAst.v arg ist.lfun } in
   eval_tactic_ist ist (CAst.make @@ TacArg (TacCall (CAst.make (Locus.ArgVar f, [Reference (Locus.ArgVar x)]))))
 
+let warn_auto_with_star = CWarnings.create ~name:"intuition-auto-with-star" ~category:"deprecated"
+    Pp.(fun () ->
+        str "\"auto with *\" was used through the default \"intuition_solver\" tactic."
+        ++ spc() ++ str "This will be replaced by just \"auto with core\" in the future.")
```

Comment:
```suggestion
        str "\"auto with *\" was used through the default \"intuition_solver\" tactic."
        ++ spc() ++ str "The default will become \"auto with core\" in the future.")
```

Is the new default "auto with core" or "auto"?  Inconsistent with the .rst description.

Reply:
Right, I wrote the changelog when this PR was different

Reply:
auto with core and auto are the same thing

Reply:
OK, but how about making the text consistent for those who aren't of that?